packages/inventory-insights/src/Constants.js

```diff
@@ -1,24 +1,19 @@
+/* eslint-disable camelcase */
 export const BASE_FETCH_URL = '/api/insights/v1/';
 
-export const FILTER_CATEGORIES = [
-    {
-        type: 'checkbox', title: 'Ansible support', urlParam: 'has_playbook', values: [
-            { label: 'Has support', value: true },
-            { label: 'No support', value: false }
+export const FILTER_CATEGORIES = {
+    has_playbook: {
+        type: 'radio', title: 'Ansible support', urlParam: 'has_playbook', values: [
+            { label: 'Ansible remediation support', value: 'true' },
+            { label: 'No Ansible remediation support', value: 'false' },
+            { label: 'All rules', value: 'all' }
         ]
-    }, {
+    }, total_risk: {
```

Comment:
Nitpick: can you put this to new line? It will look slightly better.

Reply:
resolved!